package.json

```diff
@@ -15,8 +15,8 @@
     "release-canary": "auto canary"
   },
   "peerDependencies": {
-    "react": ">=16.8.0",
-    "react-native": ">=0.59"
+    "react": ">=16.8.6",
+    "react-native": ">=0.60"
```

Comment:
This is a breaking change, would it be possible to maybe make it feature detect and only expose the new properties when running on react native >= 0.60?

Reply:
Yeah, code gets a bit messy and I had to partially disable rules of hooks. Effectively, I used the AccessibilityInfo object to infer the API version. useAccessibilityInfo hook will return early for v59. 

Personally, I'd just version this repo to match the RN version.